apps/src/templates/instructions/codeReview/CommentEditor.jsx

```diff
@@ -28,25 +25,23 @@ export default class CommentEditor extends Component {
         />
         <div
           style={{
-            ...styles.container,
-            ...styles.footer,
-            justifyContent: 'flex-end'
+            ...styles.buttonContainer
           }}
         >
-          {this.state.isEditing && (
+          {comment && (
```

Comment:
nit: can you do `comment && (both buttons)` or do you then need to surround both buttons with another tag?

Reply:
I tried but got yelled at by our version of React 😬